docs/Configuration/yaml-files.md

```diff
@@ -770,6 +770,23 @@ org_settings:
 Can only be configured for all teams (`org_settings`). To target rules to specific teams, target the
 queries referencing the rules to the desired teams.
 
+#### conditional_access
+
```

Comment:
```suggestion
_Available in Fleet Premium for Managed Cloud customers._
```

Reply:
@noahtalerman would appreciate your feedback on this wording since it's the first premium feature afaik that is limited to managed cloud customers. (Also is "Managed Cloud" a proper name or should it be lowercased?)